app/views/credits.php

```diff
@@ -2,27 +2,30 @@
 
 <p>In 2012, FrenchMozilla localization activities were merged with the wider MozFR organization. As part of this process, Transvision maintenance and developments since 1.0 were transferred to MozFR and the application is currently maintained by Pascal Chevrel.</p>
 
-<p>Here is the list of code contributors to Transvision:</p>
+<p>Here is the list of the main code contributors to Transvision:</p>
 <ul>
-    <li>Philippe Dessante</li>
+    <li>Philippe Dessante, who initiated the project</li>
     <li>Pascal Chevrel</li>
     <li>Jésús Perez</li>
     <li>Francesco Lodolo</li>
     <li>Théo Chevalier</li>
 </ul>
 
-<p>Many localizers and in particular localizers from the French team have participated in Transvision by giving feedback and making feature requests.</p>
+<p>You can find the full list on our <a href="https://github.com/mozfr/transvision/graphs/contributors">GitHub page</a>. Many localizers and in particular localizers from the French team have participated in Transvision by giving feedback and making feature requests.</p>
 
 <p>Transvision was created thanks to the following Open Source Software:</p>
 <ul>
     <li>PHP</li>
     <li>Bash for installation scripts and cron jobs</li>
     <li>a bit of Python (and specifically, the Silme library written by Zbigniew Braniecki)</li>
-    <li><a href="http://www.geany.org/">Geany</a></li>
+    <li><a href="https://getcomposer.org/">Composer</a>, a dependency manager for PHP projects</li>
+    <li><a href="https://www.atoum.org/">Atoum</a>, a unit testing framework for PHP</li>
+    <li><a href="https://www.jquery.com/">JQuery</a>, a JavaScript library</li>
+    <li><a href="https://wwww.travis-ci.com/">Travis CI</a></li>
 </ul>
 
-<p>Transvision is available on <a href="https://github.com/mozfr/transvision/">Github</a></p>
+<p>Transvision is available on <a href="https://github.com/mozfr/transvision/">GitHub</a></p>
 
 <p>Statistics about the project can be found on <a href="https://www.ohloh.net/p/transvision">Transvision Oloh page</a>.</p>
 
-<p>Transvision is hosted on the MozFR server which is a community server sponsored by Mozilla, the hosting company is <a href="http://ovh.com/">OVH</a>.</p>
+<p>Transvision is hosted on the MozFR server which is a community server sponsored and hosted by <a href="https://www.gandi.net/">Gandi</a>.</p>
```

Comment:
Suggestion to avoid the repetition of 'Hosted' in the sentence:
Transvision is a community project under the MozFR umbrella. The server is sponsored and hosted by Gandi.